presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/view/ImagesView.java

```diff
@@ -1,27 +1,30 @@
 package com.globant.equattrocchio.cleanarchitecture.mvp.view;
 
 import android.support.v7.app.AppCompatActivity;
+import android.widget.ListView;
 import android.widget.TextView;
 
 import com.globant.equattrocchio.cleanarchitecture.R;
+import com.globant.equattrocchio.cleanarchitecture.mvp.view.adapter.ListViewAdapter;
 import com.globant.equattrocchio.cleanarchitecture.util.bus.RxBus;
 import com.globant.equattrocchio.cleanarchitecture.util.bus.observers.CallServiceButtonObserver;
+import com.globant.equattrocchio.domain.model.Images;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
 import butterknife.OnClick;
 
 public class ImagesView extends ActivityView {
 
-    @BindView(R.id.tv_incoming_json) TextView tvlabel;
+    @BindView(R.id.list_view) ListView listView;
```

Comment:
Use RecyclerView instead of ListView

https://developer.android.com/guide/topics/ui/layout/recyclerview